typings.d.ts

```diff
@@ -4,13 +4,14 @@
 // are not supported
 declare module '*';
 
-declare var page_params: object;
+declare var page_params: any;
```

Comment:
Why was this change required?

Reply:
Because you can't access `page_parmas` property if the type is `object`; that's why I do an interface with `[key: string]: any;` for most of code and add todo so we can come back to it later instead of putting any. In this case though we can't and don't need to annotate `page_params` so any is fine.